.github/workflows/test.yml

```diff
@@ -141,3 +141,13 @@ jobs:
         platforms: linux/amd64,linux/arm64/v8
         tags: ${{ steps.distroless-meta.outputs.tags }}
         push: true
+
+    - name: Run Snyk to check for vulnerabilities
+      if: ${{ !contains(github.ref, 'rc') }}
+      uses: snyk/actions/docker@master
+      with:
+        image: "snowplow/elasticsearch-loader:${{ github.ref_name }}-distroless"
+        args: "--app-vulns --org=data-processing-new"
+        command: monitor
+      env:
+        SNYK_TOKEN: ${{ secrets.SNYK_TOKEN }}
```

Comment:
Just in case you're motivated: I think that it'd be nice to put the releasing + Snyk in a dedicated workflow (triggered by tagging), separated from the testing (on push to a PR)

Reply:
There's some conversation about that [over here](https://github.com/snowplow/snowplow-rdb-loader/pull/1187#issuecomment-1434807988).  Basically, doing it in a separate action adds a few extra minutes of running time -- you may or may not think that's a bad thing.  Plus, I like the directness of running the command on exactly the same image as what we push to docker hub.  i.e. build it, push it, scan the exact thing that we pushed.

Reply:
Oh I see -- that's more like [how BigQuery loader does it](https://github.com/snowplow-incubator/snowplow-bigquery-loader/tree/master/.github/workflows).  Yeah I'm happy if we make that change in more places.

Reply:
[And the collector](https://github.com/snowplow/stream-collector/tree/master/.github/workflows)